### PR TITLE
ktl-1191 feat: add ts, eslint, prettier

### PR DESCRIPTION
**Pull request by krutilov**

### CODE REVIEW

**Review by nikpachoo — APPROVED** [review]

**Comment by zoobestik** [comment]
Screenshot 2023-08-19 at 06 06 08

**Comment by zoobestik** [comment]
‼️ Looks like, it will not work after release: [`"main": "dist/playground.min.js",`](
 https://github.com/JetBrains/kotlin-playground/blob/master/package.json#L12)
Screenshot 2023-08-20 at 11 28 00

**Review by zoobestik — CHANGES_REQUESTED** [review]
🛑

**Comment by zoobestik** [comment]
Production mode has conflict:
Screenshot 2023-08-20 at 14 35 25

File: webpack.config.js [hunk 1]
```diff
@@ -84,6 +85,8 @@ module.exports = (params = {}) => {
           NODE_ENV: JSON.stringify(env),
         },
       }),
+
+      new UnminifiedWebpackPlugin()
```
Comment (nikpachoo) [inline webpack.config.js:89]: As I see it, there is no need for this plugin here.
There is a command `build:all`, which sequentially launches `npm run build` and `npm run build:production`.
I got this at the end:
Screenshot 2023-08-24 at 11 43 40

**Review by nikpachoo — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 1 above)
Comment (krutilov) [reply webpack.config.js:89]: You're right, will remove

**Review by nikpachoo — APPROVED** [review]

**Comment by nikpachoo** [comment]
I suggest putting it into the master branch.

**Pull request merged**